jbmc/regression/jbmc-strings/StringBuilderAppend03/test.java

```diff
@@ -1,10 +1,16 @@
 public class test
 {
-   public static void main(String[] args)
-   {
-      StringBuilder sb = new StringBuilder(args[0]);
-      sb.append("Z");
-      String s = sb.toString();
-      assert(s.equals("fg"));
-   }
+    public static void nondet(String stringArg, int intArg)
```

Comment:
:pick: the name suggest there should also be a `det` test

Reply:
The tests for deterministic inputs are in the StringBuilderAppend[01] folders.